connectors/v2/sullen-ural.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = $('#tracks-async').length ? '#tracks-async' : ($('.band-col-right').length ? '.band-col-right' : '.album-tracks');
```

Comment:
Could we make this a bit more readable? Having multiple ternary operators in a single statement is pretty hard to read